.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 jobs:
-  dapp_build:
+  dapp_test:
```

Comment:
We should do a build at least I think

Reply:
Disagree. CI tests (even if there are no tests) and thus builds (which will act as a check that the code is at least valid as well), so if tests are ever added, they are automatically run. CD will eventually build for the purposes of deployment/packaging.